src/trigger.cpp

```diff
@@ -0,0 +1,131 @@
+#include "avt_vimba_camera/trigger.h"
+
+namespace trigger
+{
+
+Trigger::Trigger() : vimba_system_(AVT::VmbAPI::VimbaSystem::GetInstance()), pnh_("~")
+{
+  vimba_system_.Startup();
```

Comment:
Does the Startup() call return a true or false?

Reply:
Looks like it does, I'll add a return value check.